src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Regex.cshtml.cs

```diff
@@ -29,27 +30,32 @@ public RegexModel(
 
     public IEnumerable<RegexRedirect> Items { get; set; } = Enumerable.Empty<RegexRedirect>();
 
-    private string EditItemId { get; set; }
-
     [BindProperty]
     public RegexRedirectModel RegexRedirect { get; set; }
 
+    [BindProperty]
+    public RegexRedirectModel EditRedirect { get; set; }
+
     public void OnGet()
     {
         Load();
     }
 
     public IActionResult OnPostCreate()
     {
-        if (!ModelState.IsValid)
+        if (ModelState.GetValidationState($"{nameof(RegexRedirect)}.{nameof(RegexRedirect.OldUrlRegex)}") ==
```

Comment:
What is the reason to check speciific fields only?

Reply:
ModelState dictionary contains some other fields, that are with invalid state.